src/App.js

```diff
@@ -1,42 +1,31 @@
 import "./App.css";
-import logo from "./logo.png";
+import "./css/main.css";
+
+import {  Switch,
+          Route,
+          Link, 
+          Routes} from 'react-router-dom';
```

Comment:
suggestion: use prettier to keep in one line

Reply:
I have prettier installed not sure why this happened. Anyways thanks for pointing this out.